diff --git a/common/command.c b/common/command.c
index 7171557..ce2fd2c 100644
--- a/common/command.c
+++ b/common/command.c
@@ -501,10 +501,10 @@ static int cmd_call(cmd_tbl_t *cmdtp, int flag, int argc, char * const argv[])
 	return result;
 }
 
-enum command_ret_t cmd_process(int flag, int argc, char * const argv[],
+int cmd_process(int flag, int argc, char * const argv[],
 			       int *repeatable, ulong *ticks)
 {
-	enum command_ret_t rc = CMD_RET_SUCCESS;
+	int rc = CMD_RET_SUCCESS;
 	cmd_tbl_t *cmdtp;
 
 	/* Look up command in command table */
